firefox-ios/Client/Frontend/Settings/ContentBlockerSettingViewController.swift

```diff
@@ -185,4 +202,10 @@ class ContentBlockerSettingViewController: SettingsTableViewController {
     func done() {
         settingsDelegate?.didFinish()
     }
+
+    // MARK: - ThemeApplicable
+    func applyTheme(theme: Theme) {
+        let colors = theme.colors
+        linkButton.setTitleColor(colors.actionPrimary, for: .normal)
```

Comment:
to keep consistent with other other theming, lets add `linkButton.applyTheme(theme: theme)` here instead of in its declaration and it should set the title automatically using the component.